packages/hash/frontend/src/blocks/page/ComponentView.tsx

```diff
@@ -42,13 +42,13 @@ const BLANK_PROPERTIES = {};
 
 const getChildEntity = (
   entity: DraftEntity | null | undefined,
-): DraftEntity<BlockEntity["properties"]["entity"]> | null => {
-  if (entity) {
+): DraftEntity<BlockEntity["dataEntity"]> | null => {
+  if (entity && entity.dataEntity) {
     if (!isDraftBlockEntity(entity)) {
       throw new Error("Cannot prepare non-block entity for prosemirrior");
     }
 
-    return entity.properties.entity;
+    return entity.dataEntity as DraftEntity;
```

Comment:
What is this casting needed for?

Reply:
createEntityStore assumes an optional draftId because you can use it to recreate an entity store with a new set of entities but keeping the previous draft id. Don't know if this changes anything, just want to be complete.